packages/amplify_core/lib/src/amplify_class.dart

```diff
@@ -133,7 +133,7 @@ abstract class AmplifyClass {
   static AmplifyClass? instance;
 
   /// The library version.
-  String get version => '1.0.0-next.2';
+  String get version => '1.0.0-next.6';
```

Comment:
Thanks for changing this, but why was this so far behind? Does it have to be changed manually?

Reply:
Yeah opened a separate PR to automate this